src/foam/u2/wizard/agents/DeveloperModeAgent.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```

Comment:
2022

Reply:
Things like this really hurt when I'm not on the same branch anymore